projects/plugins/boost/app/modules/optimizations/page-cache/Page_Cache.php

```diff
@@ -42,6 +42,9 @@ public function setup() {
 		add_action( 'jetpack_boost_module_status_updated', array( $this, 'handle_module_status_updated' ), 10, 2 );
 		add_action( 'jetpack_boost_critical_css_invalidated', array( $this, 'invalidate_cache' ) );
 		add_action( 'jetpack_boost_critical_css_generated', array( $this, 'invalidate_cache' ) );
+		add_action( 'update_option_' . JETPACK_BOOST_DATASYNC_NAMESPACE . '_minify_js_excludes', array( $this, 'invalidate_cache' ) );
+		add_action( 'update_option_' . JETPACK_BOOST_DATASYNC_NAMESPACE . '_minify_css_excludes', array( $this, 'invalidate_cache' ) );
+		add_action( 'update_option_' . JETPACK_BOOST_DATASYNC_NAMESPACE . '_image_cdn_quality', array( $this, 'invalidate_cache' ) );
```

Comment:
@pyronaur - is there a datasync way to get the right option to listen to for this?

Reply:
There' isn't. Bu tI think this is clean enough without adding more abstraction to datasync. If this becomes a a pattern, we can think about making this more convenient, but I think we should wait until we can see clear patterns before we do that.